src/sentry/replays/endpoints/organization_replay_count.py

```diff
@@ -57,7 +76,21 @@ class OrganizationReplayCountEndpoint(OrganizationEventsV2EndpointBase):
         }
     }
 
+    @extend_schema(
+        operation_id="Return a count of replays for a given issue or transaction id.",
+        parameters=[
+            GlobalParams.END,
+            GlobalParams.ENVIRONMENT,
+            GlobalParams.ORG_SLUG,
+            GlobalParams.START,
+            GlobalParams.STATS_PERIOD,
+            OrganizationParams.PROJECT,
+            VisibilityParams.QUERY,
+        ],
+        examples=RESPONSE_EXAMPLES,
```

Comment:
we're also missing a field here for `responses`

https://github.com/getsentry/sentry/blob/a27e8f061aa3f984f3a08af5fd684ac97974a457/src/sentry/replays/endpoints/organization_replay_index.py#L51